src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background: url(../images/star.svg) center no-repeat;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+    background: url(../images/star-active.svg) center no-repeat;
+  }
```

Comment:
Reuse stars block from previous task, it should be working for any amount of stars via changing modifier